contractsSol5/KyberMatchingEngine.sol

```diff
@@ -8,31 +8,23 @@ import "./IKyberStorage.sol";
 
 
 /*
- *   @title Kyber matching engine contract
- *   The contract provides the following actions:
- *       - return list of reserves for a trade (getTradingReserves)
- *       - stores fee accounted data for reserve types
- *       - return details about a reserve (address / id, type, isFeeAccounted)
- *       - finding the best rate (doMatch)
- *
- *       getTradingReserves() will parse hint to find if user wants specific reserves
- *
- *       doMatch() will return the index of the best reserve, having accounted for fees
- *
- */
-contract KyberMatchingEngine is
-    KyberHintHandler,
-    IKyberMatchingEngine,
-    WithdrawableNoModifiers
-{
+*   @title Kyber matching engine contract
+*   The contract provides the following actions:
+*       - return list of reserves for a trade (getTradingReserves)
+*       - stores fee accounted data for reserve types
```

Comment:
remove these two lines. not true any more

Reply:
removed